frontend/templates/troubleshooting/index.tsx

```diff
@@ -75,11 +76,18 @@ const Wiki: NextPageWithLayout<{
          <meta name="keywords" content={metaKeywords} />
          <meta name="robots" content="noindex" />,
          <link rel="canonical" href={canonicalUrl} />
+         <TagManager />
       </>
    );
 
+   console.log(
+      'matomo live url: ' +
+         process.env.NEXT_PUBLIC_MATOMO_TAG_MANAGER_CONTAINER_URL_LIVE
+   );
+
```

Comment:
Should this be remove before proceeding?

Reply:
Thanks! [d53afcf](https://github.com/iFixit/react-commerce/pull/1803/commits/d53afcf9639f8410fdee5c46e23083a3f23ced52)